src/main/resources/application.yml

```diff
@@ -201,6 +201,7 @@ offender-service:
 feature:
   flags:
     enable-cacheable-case-list: true
+    delete-hearing: true
```

Comment:
Don't we just want to set this to `true` in preprod and prod when reqjuired? I don't see the point in defining it here if we are overriding it in all environments